### DISCUSSION TITLE
InvalidFieldRelationship: Field Form has invalid relationship 'forms'.

### DISCUSSION THREAD

**Discussion by aimarsouris** [payloadcms/payload#9790] [Q&A]
Payload 3.4.0, supabase postgre, supabase storages3 + clone Github/../../templates/website, 

I get this error at pnpm dev:
node_modules/.pnpm/payload@3.4.0_graphql@16.9.0_monaco-editor@0.52.0_react-dom@19.0.0-rc-66855b96-20241106_react_re2uwt7p7ymmoimatcnocwcioe/node_modules/payload/dist/fields/config/sanitize.js (48:1) @ eval
 ⨯ InvalidFieldRelationship: Field Form has invalid relationship 'forms'.
    at Array.forEach (<anonymous>)
digest: "4014407719"

The only const which references 'forms' as a relationship is in jmm/src/app/payload/blocks/Form/config.ts (provided in the example):

export const FormBlock: Block = {
  slug: 'formBlock',
  interfaceName: 'FormBlock',
  fields: [
    {
      name: 'form',
      type: 'relationship',
      relationTo: 'forms',
      required: true,
    },
    // etc...

Any idea? Thanks!

**Comment by sylkenio** [comment]
Hey I know you posted this a while ago and might've figured it out, but if I'm not mistake, I think this type of error happens whenever the data inside of endpoints/seed wasn't seeded right. This could possibly be from the database just not being connected or your server wasn't running. Check inside of `src/endpoints/seed/index.ts` and you'll see the area where a collection called `forms` is being created by `payload.create`
